lib/common/zstd_internal.h

```diff
@@ -123,7 +123,8 @@ typedef enum { set_basic, set_rle, set_compressed, set_repeat } symbolEncodingTy
 #define MaxLit ((1<<Litbits) - 1)
 #define MaxML  52
 #define MaxLL  35
-#define MaxOff 28
+#define BasicMaxOff 28
```

Comment:
I believe I would prefer `DefaultMaxOff` here,
or anything which helps remember it is the limit for default distribution of Offset Codes.